lib/src/facts/collection.cc

```diff
@@ -25,6 +25,10 @@
 #include <yaml-cpp/yaml.h>
 #include <algorithm>
 
+#ifdef USE_WHEREAMI
```

Comment:
Since this header no longer requires the libwhereami headers, I think we can remove this guard